pkg/auth/auth.go

```diff
@@ -118,45 +149,84 @@ func (u *UserAuth) getResourcesToCheck(namespace, manifest string) ([]resource,
 	return result, nil
 }
 
-func (u *UserAuth) isAllowed(verb string, itemsToCheck []resource) error {
+func (u *UserAuth) isAllowed(verb string, itemsToCheck []resource) ([]Action, error) {
```

Comment:
if you put this behind the interface, then you can mock it to allow/disallow certain actions in tests, right?

Reply:
(related to my previous comment) Yes, I could move this to the interface but if I do that I would lose the possibility of testing this code.